<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Taschenrechner</class>
 <widget class="QMainWindow" name="Taschenrechner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Taschenrechner</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>120</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Rechenart</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>21</y>
       <width>86</width>
       <height>88</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioButtonAddition">
        <property name="text">
         <string>Addition</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonSubtraktion">
        <property name="text">
         <string>Subtraktion</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonDivision">
        <property name="text">
         <string>Division</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonMultiplikation">
        <property name="text">
         <string>Multiplikation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Berechnen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Beenden</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ergebnis</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>271</width>
      <height>23</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>32</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>Zahlensystem</string>
    </property>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>180</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioButtonDez">
        <property name="text">
         <string>Dez</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonBin">
        <property name="text">
         <string>Bin</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonHex">
        <property name="text">
         <string>Hex</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonOkt">
        <property name="text">
         <string>Okt</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>radioButtonAddition</tabstop>
  <tabstop>radioButtonSubtraktion</tabstop>
  <tabstop>radioButtonDivision</tabstop>
  <tabstop>radioButtonMultiplikation</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Taschenrechner</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonOkt</sender>
   <signal>clicked()</signal>
   <receiver>lcdNumber</receiver>
   <slot>setOctMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonHex</sender>
   <signal>clicked()</signal>
   <receiver>lcdNumber</receiver>
   <slot>setHexMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonBin</sender>
   <signal>clicked()</signal>
   <receiver>lcdNumber</receiver>
   <slot>setBinMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonDez</sender>
   <signal>clicked()</signal>
   <receiver>lcdNumber</receiver>
   <slot>setDecMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
